lib/DBus/TH.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell #-}
+module DBus.TH where
+
+import           DBus.Client
+import           DBus.Generation
+import qualified DBus.Introspection as I
```

Comment:
The qualified import of ‘DBus.Introspection’ is redundant
      except perhaps to import instances from ‘DBus.Introspection’
    To import instances alone, use: import DBus.Introspection()